src/components/NavMenu.jsx

```diff
@@ -3,17 +3,17 @@ import styles from './NavMenu.module.css';
 
 function NavMenu() {
   return (
-    <nav>
-      <ul>
-        <li>
+    <nav className={styles.nav}>
+      <ul className={styles.list}>
+        <li className={styles.listItem}>
           <NavLink
             to="/"
             className={({ isActive }) => `${styles['nav-link']} ${(isActive ? styles.selected : '')}`}
           >
             Books
           </NavLink>
         </li>
-        <li>
+        <li className={styles.listItem}>
           <NavLink
             to="/categories"
             className={({ isActive }) => `${styles['nav-link']} ${(isActive ? styles.selected : '')}`}
```

Comment:
- [ ] Please consider that there is nothing shown on the screen when I click the check status button. Kindly fix that.
![2022-03-30 23 34 32](https://user-images.githubusercontent.com/49064106/160917204-f7950dce-dabc-490c-aadc-7537924d2914.gif)